db/migrate/20180618084757_add_backup_zone_id_to_ext_management_system.rb

```diff
@@ -0,0 +1,5 @@
+class AddBackupZoneIdToExtManagementSystem < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :ext_management_systems, :backup_zone, :type => :bigint, :index => true, :after => :zone_id
```

Comment:
Remove the `:after` bit...it's not necessary.

Reply:
it makes db structure more clear, if you group logical related columns together